sound/soc/intel/boards/cht_bsw_rt5645.c

```diff
@@ -479,9 +479,18 @@ static struct snd_soc_dai_link cht_dailink[] = {
 	},
 };
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_BAYTRAIL)
+/* keep driver name user-friendly and under 16 chars with sof- prefix */
+#define CARD_RT5645_NAME "5645"
+#define CARD_RT5650_NAME "5650"
```

Comment:
I would keep rt prefix here (rt5645) for the consistency.

Reply:
yes, bad edit, will fix. thanks for spotting this.

Reply:
fixed now